server/js/sockets.js

```diff
@@ -250,6 +264,10 @@ sockets.init = function socketInit(server) {
       });
     });
 
+    socket.on('send-last-received-camera-video-feed-payloads', () => {
+      socket.emit(`camera-video-feed-status`, global.lastVideoFeedPayloads);
```

Comment:
Is there a reason we send all payloads at once for video feed status but send them separately for each device for the recording status? (Personally I think this is neater, sending everything under one topic at once)

Reply:
The `CameraRecordingStatus` component uses one payload for each component (i.e. each component is for only one device) so it would be unnecessary to send both. In contrast, `CameraRecording` addresses all the devices and thus it is suitable to send both at the same time.

I think I'm going to refactor the way retained (and status) messages are sent before I do the "is raspicam connected to mqtt" card. Probably will make a post on Slack about it.